app/components/StakeInfo/index.js

```diff
@@ -13,6 +16,76 @@ class StakeInfo extends React.Component {
     };
   }
 
+  getDefaultStyles(height) {
+    return [{ key: "output_0",style: {height: height, opacity: 0}}];
+  }
+
+  getStakeInfoDetailsStyles () {
+    const {
+      ticketPoolSize,
+      votedTicketsCount,
+      allMempoolTicketsCount,
+      missedTicketsCount,
+      ownMempoolTicketsCount,
+      revokedTicketsCount,
+      immatureTicketsCount,
+      expiredTicketsCount,
+      totalSubsidy,
+      liveTicketsCount,
+     } = this.props;
+    const { onHideStakeInfo } = this;
+    const { isShowingDetails } = this.state;
+    return [{
+      data: <StakeInfoDetails
+        {...{
+          isShowingDetails,
+          ticketPoolSize,
+          votedTicketsCount,
+          allMempoolTicketsCount,
+          missedTicketsCount,
+          ownMempoolTicketsCount,
+          revokedTicketsCount,
+          immatureTicketsCount,
+          expiredTicketsCount,
+          totalSubsidy,
+          liveTicketsCount,
+          onHideStakeInfo,
+        }}
+      />,
+      key: "output_0",
+      style: {
+        height: spring(180, {stiffness: 170, damping: 15}),
+      }
+    }];
+  }
+
+  getStakeInfoRowStyles() {
```

Comment:
I'm not a fan of this and the next one being called `*Styles` when they contain both style and data/elements.

Reply:
would it be better if it was stakeInfoRowComponent?

Reply:
I think so, yeah.